### PR TITLE
Fix export.sh

### PR DESCRIPTION
**Pull request by Chris7X**
## Description
What's changed, or what was fixed?

- Implemented a fix to handle and clean up URL inputs, ensuring they are properly formatted.
- This fix removes special characters such as carriage return (`\r`), newline (`\n`), and tab (`\t`) from the URL.

## Issues Fixed
- Fixes the issue where malformed URLs caused the error "curl: (3) URL rejected: Malformed input to a URL function". See https://www.googlecloudcommunity.com/gc/Apigee/How-to-Migrate-Proxies-and-other-data-from-Apigee-edge-to-Apigee/m-p/608135?nobounce

## Housekeeping
(please check all that apply [x], do not edit the text)
- [ ] I have run all the tests locally and they all pass.
- [ ] I have followed the relevant style guide for my changes.

## Full Repo Validation Required
(please check all that apply [x], do not edit the text)
- [ ] PR requires full pipeline run (Run for changes only by default).

**CC:** @apigee-devrel-reviewers

### CODE REVIEW

**Comment by OmidTahouri** [comment]
Hi @Chris7X -- many thanks for the fix!

A couple of things to help move this forward:

1. Please sign the CLA per [the comment above](https://github.com/apigee/devrel/pull/705#issuecomment-1739749158)
2. Please amend the commit messages in the PR to conform to [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) by prefixing `fix: ` -- e.g. `fix: sackmesser export: whitspace characters`

Out of curiousity, could you share which whitespace character was tripping you up and, if known, how the resource might have been created with that name (UI/API)?

Thanks!

File: tools/apigee-sackmesser/cmd/export/export.sh [hunk 1]
```diff
@@ -51,8 +51,9 @@ loginfo "exporting to $export_folder"
 mkdir -p "$export_folder"
 
 sackmesser list "organizations/$organization/sharedflows" | jq -r -c '.[]|. | select((. | length) > 1)' | while read -r sharedflow; do
-    # Replacing white Spaces in the sharedflow name with %20
+    # Replacing white Spaces in the sharedflow name with %20, then removes special characters
```
Comment (OmidTahouri) [inline tools/apigee-sackmesser/cmd/export/export.sh:54]: nitpick, to be more accurate: "removes **whitespace** characters"

File: tools/apigee-sackmesser/cmd/export/export.sh [hunk 2]
```diff
@@ -62,8 +63,9 @@ sackmesser list "organizations/$organization/sharedflows" | jq -r -c '.[]|. | se
 done
 
 sackmesser list "organizations/$organization/apis" | jq -r -c '.[]|.' | while read -r proxy; do
-    # Replacing white Spaces in the proxy name with %20
+    # Replacing white Spaces in the proxy name with %20, then removes special characters
```
Comment (OmidTahouri) [inline tools/apigee-sackmesser/cmd/export/export.sh:66]: nitpick, to be more accurate: "removes **whitespace** characters"

**Review by OmidTahouri — CHANGES_REQUESTED** [review]

**Comment by danistrebel** [comment]
Closing due to inactivity

**Pull request closed without merging**